<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>BulkUploadDialog</class>
 <widget class="QDialog" name="BulkUploadDialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>500</width>
    <height>450</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Bulk Upload Students</string>
  </property>
  <property name="styleSheet">
   <string notr="true">QDialog {
    background-color: white;
    font-family: "Poppins", Arial, sans-serif;
}

QLabel#titleLabel {
    color: #084924;
    font-weight: 600;
}

/* Upload area */
QFrame#uploadFrame {
    background-color: #FAFAFA;
    border: 2px dashed #C0C0C0;
    border-radius: 8px;
}

QFrame#uploadFrame:hover {
    border-color: #084924;
    background-color: #F5F5F5;
}

QPushButton#browseButton {
    background-color: #084924;
    color: white;
    border: none;
    padding: 8px 20px;
    border-radius: 6px;
    font-weight: 600;
    font-size: 10pt;
}

QPushButton#browseButton:hover {
    background-color: #0A5C2E;
}

QPushButton#downloadTemplateButton {
    background-color: transparent;
    color: #084924;
    border: 2px solid #084924;
    padding: 8px 20px;
    border-radius: 6px;
    font-weight: 600;
    font-size: 9pt;
}

QPushButton#downloadTemplateButton:hover {
    background-color: #E8F5E9;
}

QPushButton#cancelButton {
    background-color: white;
    color: #666;
    border: 2px solid #E0E0E0;
    padding: 10px 30px;
    border-radius: 8px;
    font-size: 11pt;
    font-weight: 600;
}

QPushButton#cancelButton:hover {
    background-color: #F5F5F5;
}

QPushButton#uploadButton {
    background-color: #084924;
    color: white;
    border: none;
    padding: 10px 30px;
    border-radius: 8px;
    font-size: 11pt;
    font-weight: 600;
}

QPushButton#uploadButton:hover {
    background-color: #0A5C2E;
}

QPushButton:disabled {
    background-color: #E0E0E0;
    color: #999;
}

QLabel#fileInfoLabel {
    color: #666;
    font-size: 9pt;
}

QLabel#statusLabel {
    color: #084924;
    font-weight: 600;
}</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <property name="spacing">
    <number>20</number>
   </property>
   <property name="leftMargin">
    <number>30</number>
   </property>
   <property name="topMargin">
    <number>30</number>
   </property>
   <property name="rightMargin">
    <number>30</number>
   </property>
   <property name="bottomMargin">
    <number>30</number>
   </property>
   <item>
    <widget class="QLabel" name="titleLabel">
     <property name="font">
      <font>
       <pointsize>16</pointsize>
       <bold>true</bold>
      </font>
     </property>
     <property name="text">
      <string>Bulk Upload Students</string>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QLabel" name="instructionLabel">
     <property name="text">
      <string>Upload File (eg. csv)</string>
     </property>
     <property name="styleSheet">
      <string notr="true">font-size: 10pt; color: #333;</string>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QFrame" name="uploadFrame">
     <property name="minimumSize">
      <size>
       <width>0</width>
       <height>200</height>
      </size>
     </property>
     <property name="frameShape">
      <enum>QFrame::StyledPanel</enum>
     </property>
     <property name="frameShadow">
      <enum>QFrame::Raised</enum>
     </property>
     <layout class="QVBoxLayout" name="uploadFrameLayout">
      <property name="spacing">
       <number>15</number>
      </property>
      <property name="leftMargin">
       <number>20</number>
      </property>
      <property name="topMargin">
       <number>20</number>
      </property>
      <property name="rightMargin">
       <number>20</number>
      </property>
      <property name="bottomMargin">
       <number>20</number>
      </property>
      <item>
       <widget class="QLabel" name="iconLabel">
        <property name="text">
         <string/>
        </property>
        <property name="pixmap">
         <pixmap/>
        </property>
        <property name="alignment">
         <set>Qt::AlignCenter</set>
        </property>
        <property name="styleSheet">
         <string notr="true">font-size: 48pt; color: #084924;</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLabel" name="dragDropLabel">
        <property name="text">
         <string>Drag and Drop here</string>
        </property>
        <property name="alignment">
         <set>Qt::AlignCenter</set>
        </property>
        <property name="styleSheet">
         <string notr="true">font-size: 11pt; color: #666;</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLabel" name="orLabel">
        <property name="text">
         <string>or</string>
        </property>
        <property name="alignment">
         <set>Qt::AlignCenter</set>
        </property>
        <property name="styleSheet">
         <string notr="true">font-size: 10pt; color: #999;</string>
        </property>
       </widget>
      </item>
      <item>
       <layout class="QHBoxLayout" name="browseLayout">
        <item>
         <spacer name="horizontalSpacer_2">
          <property name="orientation">
           <enum>Qt::Horizontal</enum>
          </property>
          <property name="sizeHint" stdset="0">
           <size>
            <width>40</width>
            <height>20</height>
           </size>
          </property>
         </spacer>
        </item>
        <item>
         <widget class="QPushButton" name="browseButton">
          <property name="minimumSize">
           <size>
            <width>120</width>
            <height>40</height>
           </size>
          </property>
          <property name="text">
           <string>Browse</string>
          </property>
         </widget>
        </item>
        <item>
         <spacer name="horizontalSpacer_3">
          <property name="orientation">
           <enum>Qt::Horizontal</enum>
          </property>
          <property name="sizeHint" stdset="0">
           <size>
            <width>40</width>
            <height>20</height>
           </size>
          </property>
         </spacer>
        </item>
       </layout>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <widget class="QLabel" name="fileInfoLabel">
     <property name="text">
      <string>No file selected</string>
     </property>
     <property name="alignment">
      <set>Qt::AlignCenter</set>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QLabel" name="statusLabel">
     <property name="text">
      <string/>
     </property>
     <property name="alignment">
      <set>Qt::AlignCenter</set>
     </property>
    </widget>
   </item>
   <item>
    <layout class="QHBoxLayout" name="templateLayout">
     <item>
      <spacer name="horizontalSpacer_4">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>40</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
     <item>
      <widget class="QPushButton" name="downloadTemplateButton">
       <property name="text">
        <string>Download CSV Template</string>
       </property>
      </widget>
     </item>
     <item>
      <spacer name="horizontalSpacer_5">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>40</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
    </layout>
   </item>
   <item>
    <spacer name="verticalSpacer">
     <property name="orientation">
      <enum>Qt::Vertical</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>20</width>
       <height>40</height>
      </size>
     </property>
    </spacer>
   </item>
   <item>
    <layout class="QHBoxLayout" name="buttonLayout">
     <property name="spacing">
      <number>12</number>
     </property>
     <item>
      <spacer name="horizontalSpacer">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>40</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
     <item>
      <widget class="QPushButton" name="cancelButton">
       <property name="minimumSize">
        <size>
         <width>120</width>
         <height>45</height>
        </size>
       </property>
       <property name="text">
        <string>Cancel</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="uploadButton">
       <property name="enabled">
        <bool>false</bool>
       </property>
       <property name="minimumSize">
        <size>
         <width>120</width>
         <height>45</height>
        </size>
       </property>
       <property name="text">
        <string>Upload</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>